### DISCUSSION TITLE
File privacy

### DISCUSSION THREAD

**Discussion by naftis** [uppi-fi/uppi.fi#50] [Ideas]
Allow the user to choose between "Only me", "Only with link", "Password protected"?

**Comment by naftis** [comment]
Also maybe some kind of encryption that the files actually be accessed without a logged in user session? (nice to have but maybe a selling point in the future)

**Reply by eioo** [reply]
This might be little tricky as nginx is currently hosting the files, but should be doable I think: https://docs.nginx.com/nginx/admin-guide/security-controls/configuring-jwt-authentication/

**Comment by naftis** [comment]
Could organizations share images or files securely with this tool ? (even peer to peer without our server even interrupting?)

**Reply by eioo** [reply]
Good idea, we could add some kind of "groups" feature. User could create groups and invite people to them with invite links or directly by adding username of another user.
